Makefile

```diff
@@ -371,7 +371,7 @@ install-scale-chart: install-chart-prerequisite build/toolchain/bin/helm$(EXE_EX
 # install-ci-chart will install open-match-core with pool based mmf for end-to-end in-cluster test.
 install-ci-chart: install-chart-prerequisite build/toolchain/bin/helm$(EXE_EXTENSION) install/helm/open-match/secrets/
 	$(HELM) upgrade $(OPEN_MATCH_HELM_NAME) $(HELM_UPGRADE_FLAGS) install/helm/open-match $(HELM_IMAGE_FLAGS) \
-		--set redis.ignoreLists.ttl=1000ms \
+		--set open-match-core.redis.ignoreListTTL=500ms \
```

Comment:
See also https://github.com/googleforgames/open-match/issues/1025

Though that should be a separate change.  Consider doing it before this one?

Reply:
Sure, will split this into two PRs.